amundsen_application/static/css/_layouts.scss

```diff
@@ -110,8 +112,8 @@ $resource-header-height: 84px;
 
       .editable-section,
       .metadata-section {
+        margin-top: $aside-separation-space;
```

Comment:
Unrelated Question: Did the lint work not automatically put this in alphabetical order? Thought it would happen automatically on pre-commit.

Reply:
Interesting, maybe the alphabetic ordering was part of the sass guidelines that I didn't keep.

Do you want me to find that rule and add it?

Reply:
I called it out in case this is a sign of something being broken vs the support not being there. If it's not there, I can always add it later.